docker/docker-compose.yml

```diff
@@ -1,105 +1,266 @@
-            - '2181:2181'
-        volumes:
-            - 'zookeeper_data:/var/lib/zookeeeper/data'
-        environment:
-            - ALLOW_ANONYMOUS_LOGIN=yes
-    cassandra:
-        image: 'bazaarvoice/cassandra:2.2.4'
-        networks:
-            - emodb
-        build:
-            dockerfile: cassandra-Dockerfile
-            context: .
-        environment:
-            - CASSANDRA_DC=datacenter1
-        ports:
-            - '7000:7000'
-            - '7001:7001'
-            - '7199:7199'
-            - '9042:9042'
-            - '9160:9160'
-        volumes:
-            - 'cassandra_data:/var/lib/cassandra/data'
-        depends_on:
-            - zookeeper
-    kafka:
-        image: 'confluentinc/cp-kafka:5.3.1'
-        networks:
-            - emodb
-        ports:
-            - '9092:9092'
-        volumes:
-            - 'kafka_data:/var/lib/kafka/data'
-        environment:
-            - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181
-            - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
-            - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092
-            - ALLOW_PLAINTEXT_LISTENER=yes
-            - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1
-        depends_on:
-            - zookeeper
-    emodb-web-dc1:
-        image: 'bazaarvoice/emodb-web:latest'
-        networks:
-            - emodb
-        build:
-            dockerfile: docker/Dockerfile
-            context: ../
-            args:
-                - APP_ROLE=web
-                - DATACENTER=dc1
-                - PORTS=8080 8081
-                - STARTUP_DELAY=30
+  zookeeper:
+    image: 'zookeeper:3.4.14'
+    networks:
+      - emodb
+    ports:
+      - '2181:2181'
+    volumes:
+      - 'zookeeper_data:/var/lib/zookeeeper/data'
+    environment:
+      - ALLOW_ANONYMOUS_LOGIN=yes
+    healthcheck:
+      test: ["CMD-SHELL", "echo ruok | nc 127.0.0.1 2181 || exit 1"]
+      interval: 10s
+
+  cassandra-dc1:
+    image: 'bazaarvoice/cassandra:2.1.22'
+    build:
+      context: .
+      dockerfile: cassandra-Dockerfile
+    networks:
+      - emodb
+    environment:
+      - LOCAL_JMX=no
+      - CASSANDRA_DC=datacenter1
+    ports:
+      - '7000:7000'
+      - '7001:7001'
+      - '7199:7199'
+      - '9042:9042'
+      - '9160:9160'
+      - '9164:9164'
+    volumes:
+      - 'cassandra-dc1_data:/var/lib/cassandra/data'
+    healthcheck:
+      test: ["CMD-SHELL", "nodetool status | grep UN || exit 1"]
+
+  kafka:
+    image: 'confluentinc/cp-kafka:5.3.1'
+    networks:
+      - emodb
+    ports:
+      - '9092:9092'
+    volumes:
+      - 'kafka_data:/var/lib/kafka/data'
+    environment:
+      - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+      - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092
+      - ALLOW_PLAINTEXT_LISTENER=yes
+      - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1
+    depends_on:
+      - zookeeper
+
+  emodb-blob-dc1:
+    image: 'bazaarvoice/emodb-web:6.3.1-SNAPSHOT'
```

Comment:
here and below hardcoded emodb images

Reply:
Right. It's normal since we don't use latest here.

Reply:
but what if we do new release?

Reply:
There are 2 options:
- use latest tag
- change manually for each release.

Reply:
why can't we use .env file or export tag/version?
https://docs.docker.com/compose/environment-variables/#the-env-file

Reply:
What's the difference where to define tag?

Reply:
there is no difference, we just should be able to pass it dynamically(not hardcoded)

Reply:
As I told we can use latest at this point. I'll change the docker image tag to variable and update the variable through maven later.